### PR TITLE
Completed training module "01-openapi"

### PR DESCRIPTION
**Pull request by chris-hellen**
Completed implementing the PUT pathway for the Book API. Needs review.

### CODE REVIEW

File: src/main/resources/api/api.yaml
Comment (danetsao) [inline src/main/resources/api/api.yaml:1]: This looks good.

File: src/main/java/org/folio/sample/controller/BookController.java
Comment (danetsao) [inline src/main/java/org/folio/sample/controller/BookController.java:1]: The logic looks right, however the controller should really be the management of the API interface. For the actual business logic(finding, updating, saving the book), this should be done in `BookService`. Try to implement this and lmk if you have any questions.

**Review by danetsao — APPROVED** [review]
Looks good.

**Review by chris-hellen — COMMENTED** [review]
test

**Pull request merged**